components/PageNavigation.vue

```diff
@@ -20,6 +20,31 @@
         </span>
       </SmartLink>
     </li>
+    <!-- sso links -->
+    <template v-if="enableAuthLinks">
+      <li
+        v-if="!isAuthenticated"
```

Comment:
Swap the order of the two `li` elements so that the `v-if` is: `v-if="isAuthenticated"`